From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

zhencui committed:
Add trouble shooting for no metric data

zhencui committed:
Fix some typo in the doc

zhencui committed:
Fix some style issues

itechedit committed:
edit pass: app-service-mobile-value-prop

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into app-service-mobile-value-prop

itechedit committed:
edit pass: app-service-mobile-value-prop

mumian committed:
update

mumian committed:
conflict

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - AOF Persistence

steved0x committed:
Redis Cache - persistence

Commit:
added "Disks" tile on the Storage hub page

Commit:
updated placeholder image path

Commit:
updated image path again

steved0x committed:
Redis Cache - persistence

steved0x committed:
Redis Cache - persistence

steved0x committed:
Redis Cache - persistence

steved0x committed:
Redis Cache - persistence

steved0x committed:
Redis Cache - persistence

steved0x committed:
Redis Cache - persistence

itechedit committed:
Merge branch 'master' into app-service-mobile-value-prop

mumian committed:
update

zhencui committed:
Remove en-us from the link

mumian committed:
edit

Commit:
updated disk icon

Commit:
update disk icon size

Commit:
updated disk icon size to responsive

Commit:
devices12

steved0x committed:
Redis Cache - AOF backup

steved0x committed:
Redis Cache - AOF backup

steved0x committed:
Redis Cache - AOF backup

steved0x committed:
Redis Cache - AOF backup

Commit:
Removing topic detection section due to deprecation

Removing topic detection section due to deprecation

ErikjeMS committed:
Delete azure-stack-setting-quotas.md

ErikjeMS committed:
retiring set quotas article

Commit:
move Disks tile position

v-thepet committed:
Merge pull request #20214 from LuisCabrer/patch-2

Removing topic detection section due to deprecation

rwike77 committed:
freshness updates

v-thepet committed:
Merge pull request #20206 from MarkusVi/devices12

devices12

tfitzmac committed:
fixed link

ShawnJackson committed:
Update app-service-mobile-value-prop.md

Commit:
freshness review

Commit:
Merge pull request #20246 from tfitzmac/0824fixlink

fixed link

ShawnJackson committed:
Merge pull request #19393 from itechedit/app-service-mobile-value-prop

edit pass: app-service-mobile-value-prop

rwike77 committed:
updated h3

v-thepet committed:
Merge pull request #20245 from rwike77/vmssfreshness

freshness updates

Commit:
Merge pull request #20247 from msebolt/iot-hub-devguide-device-twins

freshness review

Commit:
Merge pull request #19573 from mumian/stale04-0816

Stale04 0816

Commit:
freshness review

Commit:
Merge pull request #20254 from msebolt/iot-hub-device-management-overview

freshness review

robinsh committed:
Remove unnecessary contextual links.

carolinacmoravia committed:
P: Segmentation issue in metadata #19828

Commit:
Merge pull request #20255 from robinsh/robin-0824-contextuallinks

Remove unnecessary contextual links.

Ja-Dunn committed:
Merge pull request #20256 from carolinacmoravia/patch-363

P: Segmentation issue in metadata #19828

Commit:
freshness review

cwatson-cat committed:
Updated keyboard shortcut list

rwike77 committed:
freshness update and author change

Commit:
Merge pull request #20257 from msebolt/iot-hub-compare-event-hubs

freshness review

rwike77 committed:
author update

steved0x committed:
Redis Cache - AOF backup

Ja-Dunn committed:
Merge pull request #20258 from cwatson-cat/8-24-keyboardSC-upd

Updated keyboard shortcut list

YuriDio committed:
updating articles

Commit:
Merge pull request #20259 from rwike77/freshness2

Freshness updates and author change

Commit:
devices03

cwatson-cat committed:
Minor Azure portal TOC & index corrections suggested by pr reviewer

Commit:
freshness review

cwatson-cat committed:
Updated azure portal dashboard share access topic to address pr reviewer feedback.

Ja-Dunn committed:
Merge pull request #20261 from MarkusVi/devices03

devices03

Commit:
Merge pull request #20260 from YuriDio/AugustUpdates_5

updating articles

Ja-Dunn committed:
Merge pull request #18837 from zhencui/master

Add troubleshooting for no metric data

Commit:
devices05

Commit:
Update analysis-services-gateway.md

Commit:
update Disk verbiage and link

Ja-Dunn committed:
Merge pull request #20263 from cwatson-cat/8-24-AzurePortal-tocfixes

Minor fixes to Azure portal TOC, index, & article that were suggested by pr reviewer

Commit:
Merge pull request #20262 from msebolt/iot-hub-csharp-csharp-c2d

freshness review

Commit:
Merge pull request #20265 from MarkusVi/devices05

devices05

matt1883 committed:
Update data-lake-analytics-get-started-cli2.md

[Data Lake Analytics]
Adding content to CLI 2.0 article covering how to view pipeline and recurrence information.

matt1883 committed:
Update data-lake-analytics-get-started-cli2.md

Commit:
Merge pull request #20264 from Minewiskan/patch-2

Update analysis-services-gateway.md

Ja-Dunn committed:
Merge pull request #19652 from steved0x/sd818

Redis Cache - AOF Persistence documentation

ErikjeMS committed:
Update .openpublishing.redirection.json

Ja-Dunn committed:
Merge pull request #19778 from v-savila/1062013

for bug 1062013

KavithaJonnakuti committed:
Update sql-data-warehouse-develop-concurrency.md

Updating title of a sub-section

ErikjeMS committed:
Merge branch 'master' into master

a-tan committed:
Adding info for Recurring Job and Pipeline feature

Commit:
silly02

Commit:
Merge pull request #20271 from KavithaJonnakuti/patch-4

Update sql-data-warehouse-develop-concurrency.md

Commit:
freshness review

Commit:
Merge pull request #20273 from MarkusVi/silly02

silly02

Lisaco88 committed:
Merge pull request #20240 from ErikjeMS/master

retiring set quotas article

curtand committed:
fix picture link

Commit:
Merge pull request #20272 from a-tan/patch-1

Adding info for Recurring Job and Pipeline feature

Commit:
Merge pull request #20268 from matt1883/patch-2

[ADLA] How to view pipeline and recurrence information

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #20277 from curtand/subs0823

fix picture link

v-alje committed:
Merge pull request #20279 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #20281 from msebolt/iot-hub-live-data-visualization-in-power-bi

freshness review
